articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -0,0 +1,218 @@
+
+   - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For off-line scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group. Note that to create a Service Fabric cluster, the resource group must be empty.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+   ![Create Service Fabric cluster > Basics](images/azs-browser-create-sf-basics.png)
+
+5. In the **Network Settings** blade, enter the following information and click **OK**:
+
+   - **Service Fabric TCP Port** - Service Fabric cluster TCP gateway port to be used to connect using Service Fabric client.
+
+   - **Service Fabric HTTP Port** - Service Fabric cluster HTTP gateway port to be used to connect using Service Fabric Explorer.
+
+   - **Service Fabric Reverse Proxy Port** - Service Fabric cluster reverse proxy port
+
+   - **Custom Application Endpoints** - Custom endpoints allow for connections to applications running on this cluster. Enter endpoints separated by comma: 80,8080,8081.
+
+   - **Custom ports to open in the Network Security Group** - Custom ports to allow for the Network Security Group. Enter ports separated by comma: 3389,80,8080,8081.
+
+   ![Create Service Fabric cluster > Network Settings](images/azs-browser-create-sf-networking.png) 
+
+6. In the **Node Configuration** blade, enter the following information and click **OK**:
+
+   - **Image SKU** - The image template that each node is built from.
+
+   - **VM Size** - The size of each node.
+
+   ![Create Service Fabric cluster > Node Configuration](images/azs-browser-create-sf-node.png)
+
+7. In the **Security** blade, enter the following information and click **OK**:
+
+   - **Source Key Vault** - The resource ID of the key vault that you gathered in the previous section. This should be in the format of /subscriptions/[subscriptionid]/resourceGropus/[resourceGroupName]/providers/Microsoft.KeyVault/vaults/[vaultName].
+
+   - **Cluster Certificate URL** - This refers to the Secret Identifier of the cluster certificate which you gathered in the previous section. It should be in the format of https<!---->://[vaultEndpoint]/secrets/[secretName]/[secretVersion].
+
+   - **Cluster Certificate thumbprint** - Cluster certificate thumbprint that you gathered earlier. For example, 1742635FBCC5F9A442582516A7292523686DE3D7.
+
+   - **Server Certificate URL** - This refers to the Secret Identifier of the server certificate which you gathered in the previous section. It should be in the format of https<!---->://[vaultEndpoint]/secrets/[secretName]/[secretVersion].

```

Comment:
The Secret Identifier of the server certificate that you gathered in the previous section.

Use angled brackets <> for variables rather than square brackets []

Reply:
Fixed